apps/web/components/pages/task/description-block/editor-components/BlockButton.tsx

```diff
@@ -15,27 +17,32 @@ const BlockButton = ({
 	format,
 	icon: Icon,
 	isBlockActive,
+	className,
 }: IMarkButtonProps) => {
 	const editor = useSlate();
 
-	return (
-		<button
-			className="my-1"
-			style={{
-				background: isBlockActive(
+	const isBlockActiveMemo = useMemo(() => {
+		return (
+			isBlockActive &&
+			((format: string) => {
+				return isBlockActive(
 					editor,
 					format,
 					TEXT_ALIGN_TYPES.includes(format) ? 'align' : 'type'
-				)
-					? '#ddd'
-					: 'transparent',
-				borderRadius: '5px',
-				padding: '2px',
-				display: 'flex',
-				alignItems: 'center',
-				justifyContent: 'center',
-				transition: '0.3s',
-			}}
+				);
+			})
+		);
+	}, [editor, isBlockActive]);
+
+	return (
+		<button
+			className={clsxm(
+				className,
+				'my-1 rounded-md transition duration-300 p-[2px]',
+				isBlockActiveMemo(format)
```

Comment:
@desperado1802 I want you to use `variable` and not a function call.

Your memo function should look something like this (it should store a value and not a function)

<img width="1094" alt="image" src="https://github.com/ever-co/ever-teams/assets/81486442/0927726b-ad06-483f-aafa-1d8c4082cdae">